src/main/scala/org/hammerlab/guacamole/commands/jointcaller/VCFOutput.scala

```diff
@@ -114,100 +137,244 @@ object VCFOutput {
             .attribute("ADP", allelicDepthString(germlineEvidence.allelicDepths))
             .attribute("VAF", germlineEvidence.vaf)
-        }
-        case (TissueType.Tumor, Analyte.DNA) => {
-          val tumorEvidence = samplesEvidence.allEvidences(input.index).asInstanceOf[TumorDNASingleSampleSingleAlleleEvidence]
+
+        case (TissueType.Tumor, Analyte.DNA) =>
+          val tumorEvidence =
+            samplesEvidence
+              .allEvidences(input.index)
+              .asInstanceOf[TumorDNASingleSampleSingleAlleleEvidence]
+
           val posteriors = samplesEvidence.perTumorDnaSampleSomaticPosteriors(input.index)
+
           val thisSampleTriggered = samplesEvidence.tumorDnaSampleIndicesTriggered.contains(input.index)
+
           val sampleGenotype = samplesEvidence.parameters.somaticGenotypePolicy match {
-            case Parameters.SomaticGenotypePolicy.Presence => {
+            case Parameters.SomaticGenotypePolicy.Presence =>
               // Call an alt if there is any variant evidence in this sample and no other alt allele has more evidence.
-              val topTwoAlleles = tumorEvidence.allelicDepths.filter(_._2 > 0).toSeq.sortBy(-1 * _._2).map(_._1).take(2).toSet
+              val topTwoAlleles =
+                tumorEvidence
+                  .allelicDepths
+                  .filter(_._2 > 0)
+                  .toSeq
+                  .sortBy(-1 * _._2)
+                  .map(_._1)
+                  .take(2)
+                  .toSet
+
               topTwoAlleles match {
-                case x if x == Set(allele.ref, allele.alt) => Seq(allele.ref, allele.alt)
-                case x if x == Set(allele.ref) || x == Set(allele.alt) => Seq(x.head, x.head)
-                case _ => Seq(allele.ref, allele.ref) // fall back on hom ref
+                case x if x == Set(allele.ref, allele.alt) =>
+                  Seq(allele.ref, allele.alt)
+
+                case x if x == Set(allele.ref) || x == Set(allele.alt) =>
+                  Seq(x.head, x.head)
+
+                case _ =>
+                  Seq(allele.ref, allele.ref) // fall back on hom ref
               }
-            }
-            case Parameters.SomaticGenotypePolicy.Trigger => {
+
+            case Parameters.SomaticGenotypePolicy.Trigger =>
               // Call an alt if this sample triggered a call.
               if (thisSampleTriggered)
                 Seq(allele.ref, allele.alt)
               else
                 Seq(allele.ref, allele.ref)
-            }
+
           }
-          genotypeBuilder.alleles(JavaConversions.seqAsJavaList(sampleGenotype.map(makeHtsjdkAllele _)))
-          genotypeBuilder.attribute("RL",
-            posteriors.map(pair => "[%s]=%.8g".format(mixtureToString(pair._1), pair._2)).mkString(","))
+
+          genotypeBuilder.alleles(
+            JavaConversions.seqAsJavaList(
+              sampleGenotype.map(
+                makeHtsjdkAllele _
+              )
```

Comment:
I'm not a fan of using lines for ending parens, would rather put them on together on the last line